packages/desktop-client/src/components/sidebar/index.tsx

```diff
@@ -1,5 +1,7 @@
 import React from 'react';
 
+import { debounce } from 'lodash';
```

Comment:
Is there a specific debounce package we can import here instead of the entire lodash to avoid importing more than necessary? The bundle size info shows a 10% increase. I think there is a useDebounce hook as well. Maybe we can use that instead?

Reply:
Whoops, I went ahead and updated that